<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDialog</class>
 <widget class="QWidget" name="AddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление записи</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="button_ok">
      <property name="text">
       <string>Ок</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_cancel">
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>551</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_torg_date">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Дата торгов</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_regnum">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Код бумаги</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_price">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Текущая цена</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_gain_end">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Текущая доходность</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="line_price">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_gain_end">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>100</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_regnum">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>SU21063RMFS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SU21064RMFS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SU21065RMFS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SU21066RMFS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SU21067RMFS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SU21068RMFS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SU21069RMFS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SU21070RMFS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SU21071RMFS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SU22020RMFS</string>
    </property>
   </item>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>256</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushCalendar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать дату</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
